### ISSUE TITLE
Tbc raids/dungeons mobs hit too hard

### ISSUE THREAD

**Issue by draxan1** [gamefreedomgit/Frostmourne#1310]
**Description:**
Karazhan first pull when u going up stairs they spam spells and do a lot dmg in tanks 
hellfire ramparts mobs hit every second ability(daze/knock up/disarm etc)
botanica they hit too hard 
blood furnace working fine
bt mobs hit too hard
this is all i tested 
looks like scaling not working properly 

**How to reproduce:**
just go to any tbc dung/raid
**How it should work:**
![image](https://github.com/gamefreedomgit/Frostmourne/assets/175113484/5df87eb1-012d-4b77-bdb8-abba0f8bad98)
https://www.wowhead.com/wotlk/guide/wotlk-avoidance-calculations-how-diminishing-returns-work-21132
**Database links:**
https://wowwiki-archive.fandom.com/wiki/Damage_reduction
https://www.wowhead.com/wotlk/guide/wotlk-avoidance-calculations-how-diminishing-returns-work-21132

**Comment by BalanCV** [comment]
True, I did 10man group tempest keep yesterday and every trash pack and boss was deleting the group. Does more than wrath raids.
